sessions/talks/cltk-bringing-classical-languages-to-the-world-through-nlp.md

```diff
@@ -2,4 +2,4 @@ title: 'CLTK: Bringing Classical Languages to the World through NLP'
 subtitle:
 speaker: sourav-singh
 ---
-CLTK(Classical Language Toolkit) is a Natural Language Toolkit project which caters specifically to the Classical Languages like Classical Latin or Greek. The toolkit makes use of various Machine Learning suite like Moses or Word2Vec to create vector models for the words and sentences and detecting synonyms and is currently under active development for other classical languages like Classical Japanese and so on. I will be talking about CLTK and how this toolkit helps the archaeologists in doing reproducible research in archaeology and help in deciphering the ancient knowledge.
\ No newline at end of file
+CLTK (Classical Language Toolkit) is a Natural Language Toolkit project which caters specifically to the Classical Languages like Classical Latin or Greek. The toolkit makes use of various Machine Learning suite like Moses or Word2Vec to create vector models for the words and sentences and detecting synonyms and is currently under active development for other classical languages like Classical Japanese and so on. I will be talking about CLTK and how this toolkit helps the archaeologists in doing reproducible research in archaeology and help in deciphering the ancient knowledge.
```

Comment:
Since GitHub’s diff isn’t showing it properly: the one change here is to add a space between “CLTK” and “(Classical”.